### ISSUE TITLE
react-admin@latest points to 3.19.12

### ISSUE THREAD

**Issue by Aikain** [marmelab/react-admin#8649]
**What you were expecting:**
Latest-tag should be probably latest v4 version.

**What happened instead:**
Latest-tag points to 3.19.12

**Steps to reproduce:**
```bash
yarn add react-admin@latest
yarn why react-admin
// ...
=> Found "react-admin@3.19.12"
```

**Other information:**
`yarn upgrade-interactive --latest` show that I can "upgrade" ra-input-rich-text 4.7.4 => 3.19.12.

![image](https://user-images.githubusercontent.com/5877174/218568930-e4d880be-51dd-4b15-8f73-2153ed4c11a1.png)

**Comment by fzaninotto** [comment]
Thanks for the report. It is indeed a bug.

Unfortunately, `npm dist-tag rm react-admin latest` returns a 400 error with no explanation, so I can't "move" the latest tag to the 4.x branch without publishing a new version.

I'll do my best to publish a minor today, which should solve the issue.

**Comment by fzaninotto** [comment]
I've just published v4.8.0, which took the `latest` tag. Problem solved!

**Comment by shijiezhou1** [comment]
> I've just published v4.8.0, which took the `latest` tag. Problem solved!

Yeah it fixed now. I was so wonder why it changed the other day.

**Comment by fzaninotto** [comment]
Because we published a security fix for the 3.x branch after publishing the same fix for the 4.x branch, and for some reasons npm though this 3.x was the latest, even though semver says it's the 4.x. Go figure...
